configs/jest-config-spirit/README.md

```diff
@@ -0,0 +1,45 @@
+# jest-config-spirit
+
+> Jest configuration and preset for Spirit Design System
+
+## Getting started
+
+This preset is part of this monorepo only and thus it is accessible for every package.
+
+To install `jest-config-carbon` in your project, you will need to run the following command using [Yarn](https://yarnpkg.com/en/):
```

Comment:
I see you were inspired somewhere :D

Reply:
```suggestion
To install `jest-config-spirit` in your project, you will need to run the following command using [Yarn](https://yarnpkg.com/en/):
```

Reply:
And you might consider moving links to the bottom of the file as we do it elsewhere :)

Reply:
Fixed and moved.